language/tools/move-cli/src/experimental/cli.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{experimental, sandbox::utils::mode::Mode, Move};
+use anyhow::Result;
+use move_core_types::{
+    language_storage::TypeTag, parser, transaction_argument::TransactionArgument,
+};
+
+use structopt::{clap::arg_enum, StructOpt};
+
+#[derive(StructOpt)]
+pub enum ExperimentalCommand {
+    /// Perform a read/write set analysis and print the results for
+    /// `module_file`::`script_name`
+    #[structopt(name = "read-write-set")]
+    ReadWriteSet {
+        /// Path to .mv file containing module bytecode.
+        #[structopt(name = "module")]
+        module_file: String,
```

Comment:
How about:

```suggestion
        #[structopt(name = "module", parse(from_os_str))]
        module_file: std::path::PathBuf,
```

`analyze_read_write_set` could also be modified to take a `std::path::Path` argument for the module file. But I don't know if this is a meaningful improvement, so feel free to ignore this suggestion.